src/05-derivatives/01-introduction.md

```diff
@@ -123,43 +137,34 @@ In addition to values defined in
 | orig           | A (potentially unique) per-image space. Useful for describing the source of transforms from an input image to a target space. |
 | uri or path    | This can be used to point to a specific file.                                                                                 |
 
-### Example sidecar files
+### Examples
 
-For a NIFTI file (Single coordinate system), one could have registered the File
-on to the standard MNI305 template. The way to write the metadata of such File
-is:
+Preprocessed `bold` NIfTI file in `anat` coordinate space. Please mind that in
+this case `ReferenceMap` key is REQUIRED.
 
-```JSON
-{
-    "CoordinateSystem": "MNI305"
-}
+```Text
+sub-01/func/sub-01_task-rest_space-anat_bold.nii.gz
+sub-01/func/sub-01_task-rest_space-anat_bold.json
 ```
 
-However, it could also be the case that a nonstandard derivative of MNI305 was
-used as standard space for the File. That can be written as follows:
-
 ```JSON
 {
-    "CoordinateSystem": "MNI305",
-    "ReferenceMap": "uri or path to file"
+    "ReferenceMap": "/sub-01/anat/sub-01_desc-combined_T1w.nii.gz"
 }
 ```
 
-Some derivatives such as CIFTI Files allow for multiple coordinate systems. Such
-possibility is enabled by using lists of spaces and references:
+Participant cortical thickness GIFTI file in `fsnative` coordinate space. Please
+mind that in this case `ReferenceMap` key is REQUIRED.
 
-```JSON
-{
-    "CoordinateSystem": ["MNI305", "fsLR32k"]
-}
+```Text
+sub-01/anat/sub-01_hemi-L_space-fsnative_thickness.shape.gii
+sub-01/anat/sub-01_hemi-L_space-fsnative_thickness.json
 ```
 
-Differing references of the same spaces with respect to the above example can be
-expressed as follows:
-
 ```JSON
 {
-    "CoordinateSystem": ["MNI152Lin", "fsLR164k"]
+    "CoordinateSystem": "fsnative",
```

Comment:
are we keeping this, or simply using `space`?

Reply:
good catch